data/Application.css

```
@@ -26,8 +26,7 @@
 }
 
 vte-terminal {
-    padding-left: 4px;
-    padding-right: 4px;
+    padding: 0 12px;
```

How about this? Makes it look more uniform and less lopsided. :smile: 

```suggestion
    padding: 12px 12px 0 12px;
```

---

So this still has the weird scrolling problem. I think just applying to the sides for now is fine unless/until we can sort out the undershoot.

---

@danrabbit I think 0 and 12px feels too lopsided, so unless/until we can sort out some sort of undershoot or other way to handle that, how about `0 9px`? Still a notable improvement over master.

```suggestion
    padding: 0 9px;
```